x-pack/filebeat/input/awss3/collector_test.go

```diff
@@ -58,11 +58,25 @@ func (m *MockS3Client) GetObjectRequest(input *s3.GetObjectInput) s3.GetObjectRe
 
 func TestGetRegionFromQueueURL(t *testing.T) {
 	queueURL := "https://sqs.us-east-1.amazonaws.com/627959692251/test-s3-logs"
-	regionName, err := getRegionFromQueueURL(queueURL)
+	regionName, err := getRegionFromQueueURL(queueURL, "")
 	assert.NoError(t, err)
 	assert.Equal(t, "us-east-1", regionName)
 }
 
+func TestGetRegionFromQueueURLOtherEndpoint(t *testing.T) {
```

Comment:
Thank you for adding the test! Could you actually combine these two tests together using the `[]struct` please? You can follow `TestHandleMessage` for syntax.

Reply:
Will do.

Reply:
done